### PR TITLE
Adds AAF_RSE

### PR DESCRIPTION
**Pull request by RowanSkie**
Adds Loverslab mod: AAF RSE. Nothing else.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -9483,7 +9483,10 @@ plugins:
     clean:
       - crc: 0x0e2861fa
         util: FO4Edit v3.2.1
-
```
Comment (Freso) [inline masterlist.yaml]: Is there a reason you're removing this newline? Seems like this line would still be good to (visually) separate the alternate start mods from the `FO4Merged.esp` one.

**Review by Freso — CHANGES_REQUESTED** [review]

**Comment by Freso** [comment]
(Other than that comment, it looks good. Well done. :))

File: masterlist.yaml (hunk 1 above)
Comment (RowanSkie) [reply masterlist.yaml]: I'll get right into it. Must've been an overlooked by me.

**Review by Freso — APPROVED** [review]

**Comment by Freso** [comment]
Merged in with https://github.com/loot/fallout4/commit/bcc8c7accf8e3a4347b03a0b68e2049685653f26. Thanks! :)

**Pull request closed without merging**